woocommerce/wc-admin
# Move non-reset SCSS to new files, add folder structure

This PR creates a new file for any "global" scss. Things like css variable definitions or `@keyframes` should live here rather than in the [files included from webpack](https://github.com/woocommerce/wc-admin/blob/master/webpack.config.js#L70-L73), since this is still written to the final CSS file, and having it in the webpack includes would duplicate it for every component scss file.

While doing that, I noticed that there's some embed-pages specific code in the reset file, so I decided to take the opportunity to add some structure to our CSS. I've moved the notice styles in the activity-panel scss, and the embed styles are now in a new file, `_embed.scss`, which is only included in embed pages.

The new folder structure is 

- `/abstracts/*` – this contains all mixins/variables/etc, stuff that won't appear in compiled CSS
- `/shared/*` – these are the styles that will appear in compiled CSS - the reset, gutenberg components import, etc
- `/_index.scss` – this file is used for the app's CSS, and is imported in `index.js` only
- `/_embedded.scss` – this file is used for the embed pages CSS, and is imported in `embedded.js` only

**To test**

- Check that CSS still compiles
- Visually check over each page, especially the embed pages, to make sure no style regressions have appeared
